### PR TITLE
pause when out of view functionallity

### PR DESCRIPTION
**Pull request by almoghaimo**

### CODE REVIEW

File: Sources/Helpers/Extensions.swift [hunk 1]
```diff
@@ -72,6 +72,13 @@ extension UIColor {
     }
 }
 
+extension UIView {
+    var allSubviews: [UIView] {
+        return subviews + subviews.flatMap { $0.allSubviews }
+    }
+
+}
```
Comment (HasanSa) [inline Sources/Helpers/Extensions.swift]: combine the `UIView` extensions

File: Sources/Helpers/Constants.swift [hunk 2]
```diff
@@ -68,5 +68,6 @@ struct Constants {
         static let resize = "apester_resize_unit"
         static let height = "height"
         static let width = "width"
+        static let validateStripViewVisibity = "validateStripViewVisibity"
```
Comment (HasanSa) [inline Sources/Helpers/Constants.swift]: shouldn't you update the function name to be something like `validateUnitViewVisibity`

Comment (almoghaimo) [reply Sources/Helpers/Constants.swift]: I think @OmriKeret says it should stay as is, but I'll double check with him.

Comment (almoghaimo) [reply Sources/Helpers/Constants.swift]: I'll update it tx.

**Pull request closed without merging**